index.d.ts

```diff
@@ -1,11 +1,17 @@
+import { LaunchOptions } from "puppeteer";
```

Comment:
You cannot import in definition files, you'll have to use and inline import that is resolved during typescript compilation
```suggestion
type LaunchOptions = import("puppeteer").LaunchOptions;
```